### DISCUSSION TITLE
Issues with Recording

### DISCUSSION THREAD

**Discussion by tech2mar** [blakeblackshear/frigate#5577] [General]
Ayo, frigate people, I have some error trying to config the cams to record... 😳 Something with a stream that doesn't work...

**Comment by NickM-27** [comment]
No one can help if you don't include the error you're seeing and your current config

**Reply by tech2mar** [reply]
Sorry for the bad explanation :



[2023-02-23 22:58:55] frigate.http ERROR : No recordings found for the requested time range


"The media could not be loaded, either because the server or network failed or because the format is not supported."


Config : used 

https://docs.frigate.video/configuration/record/

**Reply by NickM-27** [reply]
Please paste your config file.

**Reply by tech2mar** [reply]
https://0bin.net/paste/eL3waGEP#MqQy6VSJEi42rCxmrJim9wzhbwXgDqi+OryppUqRGXG

In this version the formatting is incorrect, my frigate starts correctly... But I can't have any clips..

**Reply by NickM-27** [reply]
do you see recordings segments being added in your recordings folder?

**Reply by tech2mar** [reply]
You mean some folder on my HASS ?
Now that you say, wehre would frigate save the recordings?

**Reply by tech2mar** [reply]
After a reboot, and some patience it works and I can find the clips also on the storage.

Thanks

**Reply by tech2mar** [reply]
Is it possible to have that config on the global settings? So that all camera is doing the same thing? Without having to config all camera individually?

**Reply by NickM-27** [reply]
> Is it possible to have that config on the global settings? So that all camera is doing the same thing? Without having to config all camera individually? 

Yes it can be defined globally

**Reply by tech2mar** [reply]
oh great thanks, i hope this project will grow up. this is one of my favorite
